tidy3d/components/medium.py

```diff
@@ -131,6 +131,30 @@ def eps_diagonal(self, frequency: float) -> Tuple[complex, complex, complex]:
         eps = self.eps_model(frequency)
         return (eps, eps, eps)
 
+    def eps_comp(self, row: Axis, col: Axis, frequency: float) -> complex:
+        """Single component of the complex-valued permittivity tensor as a function of frequency.
+
+        Parameters
+        ----------
+        row : Axis
+            Component's row in the permittivity tensor.
+        col : Axis
+            Component's column in the permittivity tensor.
+        frequency : float
+            Frequency to evaluate permittivity at (Hz).
+
+        Returns
+        -------
+        complex
+           Element of the relative permittivity tensor evaluated at ``frequency``.
+        """
+
+        # This only needs to be overwritten for anisotropic materials
+        if row == col:
+            eps = self.eps_model(frequency)
+            return eps
+        return 0
```

Comment:
let's return a complex number here `0j` just to make sure it's consistent with the return type.

Reply:
fixed